pkg/utils/configs/manifest.go

```diff
@@ -42,7 +45,7 @@ func (manager *ManifestManager) GetDefaultManifestWithEnv() Manifest {
 	}
 }
 
-func (manager *ManifestManager) CreateFile(m Manifest) error {
+func (manager *ManifestManager) CreateFile(m SkupperManifest) error {
```

Comment:
I think this method can be removed, because no function is calling it.

Reply:
done